include/nccl_ofi_param.h

```diff
@@ -239,14 +239,27 @@ OFI_NCCL_PARAM_INT(disable_dmabuf, "DISABLE_DMABUF", 0);
 OFI_NCCL_PARAM_UINT(min_stripe_size, "MIN_STRIPE_SIZE", (64 * 1024));
 
 /*
- * Minimum bounce buffers posted per rail. The plugin will attempt to post
+ * Minimum ctrl recv buffers posted per rail. The plugin will attempt to post
+ * more buffers if we dip below this threshold, allocating new buffers if needed.
+ */
+OFI_NCCL_PARAM_INT(rdma_min_posted_ctrl_recv_buffers, "RDMA_MIN_POSTED_CTRL_RECV_BUFFERS", 64);
```

Comment:
Shouldn't this be a function out max outstanding requests, which today we have at 128?

Reply:
That's a bit tricky, because the outstanding requests max (`NCCL_OFI_MAX_REQUESTS`) is
1. A per-communicator value, whereas this parameter is a per-endpoint value, so we'd have to multiply this parameter by the number of communicators, which we don't know in advance
2. Much higher than NCCL actually keeps in flight typically, so setting this param to `NCCL_OFI_MAX_REQUESTS x num_comms` will be overkill.

That said, we may need to tune this value, but the current value in this PR is already significantly higher than what is in master today (which is 16-32 per rail, and shared between eager and ctrl recv buffers).